sdk/containerregistry/azure-containerregistry/tests/test_container_registry_client.py

```diff
@@ -19,12 +19,14 @@
 from azure.identity import DefaultAzureCredential
 
 from testcase import ContainerRegistryTestClass
+from constants import TO_BE_DELETED
```

Comment:
is this constant somethign you want exposed to users? I'm assuming not (which is how it currently is), but just double checking

Reply:
No it is not, it's just so I have a consistent registry imported. I kept misspelling it and eventually made it a constant.